regression-test/suites/query_p0/sql_functions/string_functions/test_split_part.groovy

```diff
@@ -27,4 +27,6 @@ suite("test_split_part") {
     """
     exception "[RUNTIME_ERROR]Argument at index 3 for function split_part must be constant"
   }
+
+  qt_1 "select split_part(k8, '1', 1), k8, split_part(concat(k8, '12'), '1', 1) from test_query_db.test order by k8 limit 2;"
```

Comment:
Does this regression test related with the issue? It does not have any pushdown predicates.

Reply:
topn generated a runtime predicate, a newly supported feature in pr #15558